octoprint_bambu_printer/printer/states/paused_state.py

```diff
@@ -44,7 +44,7 @@ def _sendPaused(self):
         )
         paused_timer.start()
 
-    def start_new_print(self):
+    def start_resume_print(self):

```

Comment:
"start_resume_print" does not actually makes sense as state function naming.
I would rather name the gcode handler for starting the print "start_resume" and leave state functions as separate "start" and "resume" functions.
I will submit my version in next commits